kumascript/src/render.js

```diff
@@ -67,6 +67,10 @@ async function render(
   renderPrerequisiteFromURL,
   { templates = null } = {}
 ) {
+  pageEnvironment.slug = pageEnvironment.slug.replace(
+    /^(orphaned)|(conflicting)\//,
+    ""
+  );
```

Comment:
Clever!

Reply:
Hopefully, once those pages have been built once and uploaded to S3, we can delete them from the source (`translated-content`) and also delete this line of code. 
Right? Are we intending to keep any of those pages around?

Reply:
Note-to-self;
```
MOZILLA/MDN/translated-content  main ✔
▶ fd index.html | rg orphaned | wc -l
    1138

MOZILLA/MDN/translated-content  main ✔
▶ fd index.html | rg conflicting | wc -l
    1056
```